src/app/settings/categories/agencies/agencies.component.html

```diff
@@ -0,0 +1,89 @@
+<div>
+    <legend style="margin-bottom:25px;margin-top:0;">Agencies</legend>
+</div>
+<button [hidden]="loggedInRole !== 'Administrator'" type="button" class="btn-blue" (click)="showNewAgencyForm()"><i class="fa fa-plus"
```

Comment:
This is probably copied from the other settings components, but since the settings component is only available for admins, we can remove this check for loggedinrole == admin (originally, we had the settings available for any logged in users)